src/domain/CreateEventStreamRecordRepository.ts

```diff
@@ -11,3 +14,20 @@ export type CreateEventStreamRecord = AuditRecord & {
 };
 
 export type CreateEventStreamRecordRepository = Repository<CreateEventStreamRecord>;
+
+export const isCreateEventStreamRecord = (record: AllRecord): O.Option<CreateEventStreamRecord> =>
+  record.type === 'CreateEventStreamRecord' ? O.some(record) : O.none;
+
+export const hasTimelineEventTypeToTimeline = (record: CreateEventStreamRecord) =>
+  record.input.body.eventType === EventTypeEnum.TIMELINE;
+
+export const isSuccessfulResponse = (record: CreateEventStreamRecord) => record.output.statusCode === 200;
```

Comment:
🤔 I'm not sure if the method `isSuccessfulResponse` is worth, it is used only in one place and somehow tested multiple times. Am I a bad human being (😈) if I think we should remove it? WDYT?

Reply:
Yeah, agree. Done!